src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -681,10 +682,10 @@ sub check_dither {
             push @{$self->{warn}}, $warn;
         }
     }
-
-
     # Check for standard and large dither based solely on backstop/history values
     if (($bs_val eq 'ENAB') and (defined $dither->{ampl_y} and defined $dither->{ampl_p})){
+        $self->{cmd_dither_y_amp} = $dither->{ampl_y};
+        $self->{cmd_dither_z_amp} = $dither->{ampl_p};
```

Comment:
I think it is appropriate and makes sense to only save out these amplitudes if dither is enabled (which is what this is doing).